src/sass/partials/components/_fourohfour.scss

```diff
@@ -0,0 +1,16 @@
+.fourohfour {
```

Comment:
Instead of `fourohfour` (which is hard to read), it should either be `four-oh-four` or, probably better `not-found`. Update the name of the file accordingly